config/initializers/dalli.rb

```diff
@@ -0,0 +1,22 @@
+module Memcached
+  def self.current
+    @current
+  end
+
+  def self.current=(current)
+    @current = current
+  end
+end
+
+if ENV["MEMCACHIER_SERVERS"]
+  Memcached.current = Dalli::Client.new(
+    (ENV["MEMCACHIER_SERVERS"] || "").split(","),
+    username: ENV["MEMCACHIER_USERNAME"],
+    password: ENV["MEMCACHIER_PASSWORD"],
+    failover: true,
+    socket_timeout: 1.5,
+    socket_failure_delay: 0.2,
+  )
+else
+  Memcached.current = Dalli::Client.new("127.0.0.1:11211")
+end
```

Comment:
I think we should use `Rails.cache` instead of creating a new client instance on boot.